packagedb/api.py

```diff
@@ -709,6 +509,196 @@ class PackageSetViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = PackageSet.objects.prefetch_related('packages')
     serializer_class = PackageSetAPISerializer
 
+class CollectViewSet(viewsets.ViewSet):
```

Comment:
Are we also going to have an endpoint that collects and returns package data for a single purl in the `Collect` endpoint, or is that functionality going to the `Package` endpoint

Reply:
We're keeping fetch and return single purl in `collect`  endpoint

- Fetching and returning single purl - GET: `/api/collect?purl=pkg;npm/package-name@version`
- Bulk Indexing - POST: `/api/collect/index_packages/`
- Bulk Reindexing - POST: `/api/collect/reindex_packages/`